src/components/ReportActionItem/ReportPreview.js

```diff
@@ -232,6 +237,15 @@ function ReportPreview(props) {
                                 style={[styles.requestPreviewBox]}
                             />
                         )}
+                        {isReportDraft && (
+                            <Button
+                                medium
+                                success={props.iouReport.isOwnPolicyExpenseChat}
```

Comment:
Its the policy expense chat in this case chat report which has this property. Could we also access it safely?
```suggestion
                                success={props.chatReport.isOwnPolicyExpenseChat}
```